tests/pdctl/config/config_test.go

```diff
@@ -200,3 +200,54 @@ func (s *configTestSuite) TestConfig(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(strings.Contains(string(output), "config item not found"), IsTrue)
 }
+
+func (s *configTestSuite) TestConfigDefaultStoreLimit(c *C) {
+	// We renamed store-balance-rate to default-store-limit, new tests
```

Comment:
Good comment. Our other tests should also learn from you to add the necessary comments.

Reply:
Thanks :)